internal/prompt/editor.go

```diff
@@ -1,68 +1,86 @@
 package prompt
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 	"os/exec"
-	"strings"
+	"runtime"
+
+	"github.com/kballard/go-shellquote"
+)
+
+const (
+	defaultEditor = "vim"
 )
 
-const defaultEditor = "vim"
+var (
+	bom        = []byte{0xef, 0xbb, 0xbf}
+	cliEditors = []string{"vi", "vim", "nano"}
```

Comment:
```suggestion
	cliEditors = []string{"vi", "vim", "nano","micro"}
```
Add [micro](https://github.com/zyedidia/micro)